coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -615,6 +627,34 @@ func (server *Server) CompleteJob(ctx context.Context, completed *proto.Complete
 			}
 		}
 
+		server.Logger.Info(ctx, "template import !", slog.F("template_import", jobType.TemplateImport.String()))
```

Comment:
This message could be more informative. Maybe: `completed template import` or something.

Reply:
Oh, my bad. That's for debugging purposes :)

Reply:
I kinda figured ;p